cockpit/webpack.config.js

```diff
@@ -156,13 +157,6 @@ if (!production) {
 
 /* Only minimize when in production mode */
 if (production) {
-    plugins.unshift(new webpack.optimize.UglifyJsPlugin({
-        beautify: true,
-        compress: {
-            warnings: false
-        },
-    }));
-
```

Comment:
BTW: Why do you want to remove this?

Reply:
I replaced with TerserJSPlugin which we also use in cockpit-podman.
I have nothing against this plugin, but it would be ideal if the webpack config would be same, if not identical between cockpit plugins.
I will revert this change - since I noticed that cockpit itself also uses the uglify-js one.